src/poller_base.hpp

```diff
@@ -130,6 +130,8 @@ class poller_base_t
     void add_timer (int timeout_, zmq::i_poll_events *sink_, int id_);
     void cancel_timer (zmq::i_poll_events *sink_, int id_);
 
+    virtual int wait (int timeout) = 0;
```

Comment:
This method deserves some documentation. What should it wait for? What do the return values mean?

Reply:
Totally agree. This function behave like select/poll/...